instant-xml/src/parse.rs

```diff
@@ -1,48 +1,152 @@
-use xmlparser::{ElementEnd, Token};
-
-use super::Error;
-
-impl<'a> Parse for Option<Result<xmlparser::Token<'a>, xmlparser::Error>> {
-    fn element_start(self, ns: Option<&str>, tag: &str) -> Result<(), Error> {
-        match self {
-            Some(Ok(Token::ElementStart { prefix, local, .. })) => {
-                let prefix_ns = prefix.as_str();
-                let (has_prefix, expect_prefix) = (!prefix_ns.is_empty(), ns.is_some());
-                if has_prefix != expect_prefix {
-                    return dbg!(Err(Error::UnexpectedValue));
-                }
+use crate::{Error, Result};
```

Comment:
Please separate imports in three blocks separated by a newline

* `std` and/or `core`
* external crates (xmlparser)
* crate-internal symbols

Reply:
done